automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -130,9 +130,13 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   }
 
   def clickAndReadSuggestions(field: String): Seq[String] = {
-    click on testId(s"$field-text-input")
-    val elements = findAll(xpath(s"//*[@data-test-id='$field-suggestions']/*/li"))
-    elements.map(_.text).toSeq
+    val dataTestId = s"$field-text-input"
+    click on testId(s"$dataTestId")
+    // wait for dropdown to become expanded
+    await condition (find(testId(s"$dataTestId")).exists(_.underlying.getAttribute("aria-expanded") == "true"), 10)
+    // wait for dropdown to contain at least one WebElement
+    await condition (findAll(xpath(s"//*[@data-test-id='$field-suggestions']/*/li")).map(elem => elem.isDisplayed).nonEmpty)
+    findAll(xpath(s"//*[@data-test-id='$field-suggestions']/*/li")).map(_.text).toSeq
```

Comment:
wait for dropdown to become expanded by waiting for attribute `aria-expanded` to be `true`. then find all WebElement inside dropdown. verify each WebElement is `isDisplayed`. forcing stale exception to happen if any WebElement isn't ready for user interaction. catch exception then retry.
```
<input type="search" autocomplete="off" role="combobox" aria-autocomplete="list" aria-owns="react-autowhatever-autosuggest123" aria-expanded="true" data-test-id="test.hello.name-text-input" placeholder="Optional" value=""
```